docs/usage/PostMigrationCleanup.md

```diff
@@ -0,0 +1,50 @@
+# Post-migration cleanup
+
+If you've finished migrating your apps with Crane and you want to cleanly remove Crane and associated migration resources from the cluster, follow these steps:
+
+
+## Clean up steps
+
+### For _control clusters_ where mig-controller is installed:
+
+1. Login to the cluster
+1. Set `spec.closed: true` on all MigPlans to run cleanup routines for each plan. This will remove migration related resources spread across the cluster.
+    ```sh
+    for migplan in $(oc get migplan -n openshift-migration -o jsonpath='{.items[*].metadata.name}'); do oc -n openshift-migration patch migplan $migplan --type=json --patch '[{ "op": "add", "path": "/spec/closed", "value": "true" }]'; done
```

Comment:
In the above, `"true"` needs to be `true`. When I tried the above as-is, I got the following error:
```
The MigPlan "change-namespace-dvm" is invalid: spec.closed: Invalid value: "string": spec.closed in body must be of type boolean: "string"
```
When I removed the quotes around `true`, it worked as expected.

Reply:
Thanks for catching this. My OCP 4 cluster stopped responding yesterday in the middle of writing this doc so hadn't actually run this full command, I wondered if the quotes would need to go.

Reply:
Fixed